_config/cli.yml

```diff
@@ -0,0 +1,13 @@
+---
+Name: cli-config
+---
+SilverStripe\Core\Injector\Injector:
+  Symfony\Contracts\EventDispatcher\EventDispatcherInterface.sake:
+    class: 'Symfony\Component\EventDispatcher\EventDispatcher'
```

Comment:
Required for Sake event dispatcher, which allows devs to hook into symfony/console events if they want to